kie-wb-common-dmn/kie-wb-common-dmn-client/src/test/java/org/kie/workbench/common/dmn/client/editors/expressions/jsinterop/util/ExpressionModelFillerTest.java

```diff
@@ -0,0 +1,334 @@
+        assertThat(decisionTableExpression.getOutput()).isNotNull();
+        assertThat(decisionTableExpression.getOutput()).hasSize(1);
+        assertThat(decisionTableExpression.getOutput()).first().extracting(OutputClause::getName).isEqualTo(outputColumn);
+        assertThat(decisionTableExpression.getOutput()).first().extracting(outputClause -> outputClause.getTypeRef().getLocalPart()).isEqualTo(outputDataType);
+        assertThat(decisionTableExpression.getRule()).isNotNull();
+        assertThat(decisionTableExpression.getRule()).hasSize(1);
+        assertThat(decisionTableExpression.getRule()).first().extracting(rule -> rule.getInputEntry().size()).isEqualTo(1);
+        assertThat(decisionTableExpression.getRule()).first().extracting(DecisionRule::getInputEntry).extracting(inputEntry -> inputEntry.get(0).getText().getValue()).isEqualTo(inputValue);
+        assertThat(decisionTableExpression.getRule()).first().extracting(DecisionRule::getOutputEntry).extracting(outputEntry -> outputEntry.get(0).getText().getValue()).isEqualTo(outputValue);
+        assertThat(decisionTableExpression.getRule()).first().extracting(DecisionRule::getAnnotationEntry).extracting(annotationEntry -> annotationEntry.get(0).getText().getValue()).isEqualTo(annotationValue);
+        assertThat(decisionTableExpression.getComponentWidths()).isNotNull();
+        assertThat(decisionTableExpression.getComponentWidths()).hasSize(4);
+        assertThat(decisionTableExpression.getComponentWidths()).element(1).isEqualTo(inputWidth);
+        assertThat(decisionTableExpression.getComponentWidths()).element(2).isEqualTo(outputWidth);
+        assertThat(decisionTableExpression.getComponentWidths()).element(3).isEqualTo(annotationWidth);
+    }
+
+    private ContextEntryProps buildContextEntryProps() {
+        return new ContextEntryProps(new EntryInfo(ENTRY_INFO_NAME, ENTRY_INFO_DATA_TYPE), new LiteralExpressionProps("Nested Expression", BuiltInType.UNDEFINED.asQName().getLocalPart(), ENTRY_EXPRESSION_CONTENT, null));
+    }
+
+    private void assertEntryWidths(final Collection<Double> componentWidths) {
+        assertThat(componentWidths).isNotNull();
+        assertThat(componentWidths).hasSize(3);
+        assertThat(componentWidths).element(1).isEqualTo(ENTRY_INFO_WIDTH);
+        assertThat(componentWidths).element(2).isEqualTo(ENTRY_EXPRESSION_WIDTH);
+    }
+
+    private void assertFormalParameters(FunctionDefinition functionExpression) {
+        assertThat(functionExpression.getFormalParameter()).isNotNull();
+        assertThat(functionExpression.getFormalParameter()).hasSize(1);
+        assertThat(functionExpression.getFormalParameter()).first().extracting(param -> param.getValue().getValue()).isEqualTo(PARAM_NAME);
+        assertThat(functionExpression.getFormalParameter()).first().extracting(param -> param.getTypeRef().getLocalPart()).isEqualTo(PARAM_DATA_TYPE);
+    }
```

Comment:
Generally I think, we overuse `assertThat` what cause some assertions less readable, one example how come can be simplified:
```
    private void assertFormalParameters(FunctionDefinition functionExpression) {
        assertThat(functionExpression.getFormalParameter())
                .isNotNull()
                .hasSize(1)
                .first()
                .satisfies(param -> {
                    assertThat(param.getValue().getValue()).isEqualTo(PARAM_NAME);
                    assertThat(param.getTypeRef().getLocalPart()).isEqualTo(PARAM_DATA_TYPE);
                });
    }
```